src/osd/OpRequest.h

```diff
@@ -13,15 +13,7 @@
 
 #ifndef OPREQUEST_H_
 #define OPREQUEST_H_
-#include <sstream>
-#include <stdint.h>
-#include <vector>
```

Comment:
I don't think we should remove this header. `OpRequest.h` references `std::vector`. the same applies to `Message` and `Message.h`. in that case, we could use a forward declaration though.